app/javascript/src/Settings/components/OidcFieldset.jsx

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import React, {useState} from 'react'
+import { FormFieldset, FormLegend } from 'Form'
+import { Checkbox } from '@patternfly/react-core'
+import { FormCollection, TypeItemCombo } from 'Settings/components/Common'
+import type { TypeItemProps, FieldGroupProps } from 'Settings/types'
+
+const Basics = (props: TypeItemProps) => (
+  <TypeItemCombo {...props} legend='OIDC BASICS' inputType='url' />
+)
+
+const JsonWebToken = (props: TypeItemProps) => (
+  <TypeItemCombo {...props} legend='JSON Web Token (JWT) Claim with ClientID' inputType='text' />
+)
+
+const FlowItem = ({name, label, checked}: FieldGroupProps) => {
+  const [ isChecked, setIsChecked ] = useState(checked)
+  const onChange = (check) => setIsChecked(check)
```

Comment:
```suggestion
  const onChange = setIsChecked
```

Reply:
This is the same case as in https://github.com/3scale/porta/pull/1315#discussion_r340999672, look: https://www.patternfly.org/v4/documentation/react/components/checkbox/#checkbox-props. I'll make it explicit adding the second (not used) argument

Reply:
Sorry, I'm repeating myself, forgot about https://github.com/3scale/porta/pull/1315#discussion_r340999672